geometry_msgs/Twist desired_pos
float64[] q_in
---
float64[] q_out
